parsimonious/grammar.py

```diff
@@ -296,64 +298,74 @@ def __init__(self, custom_rules=None):
         """
         self.custom_rules = custom_rules or {}
 
-    def visit_parenthesized(self, parenthesized, (left_paren, _1,
-                                                  expression,
-                                                  right_paren, _2)):
+    def visit_parenthesized(self, parenthesized, _a):
```

Comment:
As I said on #79, it's a shame that Python 3 removed support for tuple unpacking in arg lists, as it's something I lean heavily upon to make Parsimonious's visitors easier to read. That's one advantage of 2to3 over six: Python 2 people still get the more readable code. But I'm willing to sacrifice this for easier development under 3.

However, unless there's a dominant convention that I'm unaware of, can we pick a more descriptive name for the new arg than `_a`?